pkgs/by-name/sk/skeditor/package.nix

```diff
@@ -0,0 +1,34 @@
+{
+  lib,
+  buildDotnetModule,
+  fetchFromGitHub,
+  dotnetCorePackages,
+}:
+buildDotnetModule rec {
+  pname = "skeditor";
+  version = "2.8.1";
+
+  src = fetchFromGitHub {
+    owner = "skeditorteam";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-ojE276nonX52UTjvdKL5mX8dj3MNElqlR1A/c0vT9WE=";
+  };
+
+  projectFile = "SkEditor/SkEditor.csproj";
+  executables = [ "SkEditor" ];
+  nugetDeps = ./nuget-deps.nix;
+
+  dotnet-sdk = dotnetCorePackages.sdk_8_0;
+  dotnet-runtime = dotnetCorePackages.runtime_8_0;
+
+  passthru.updateScript = ./update.sh;
+
+  meta = with lib; {
+    description = "SkEditor is an app for editing Skript files";
+    homepage = "https://github.com/SkEditorTeam/SkEditor";
+    changelog = "https://github.com/SkEditorTeam/SkEditor/releases/tag/v${version}";
+    licence = licenses.mit;
+    maintainers = with maintainers; [ eveeifyeve ];
```

Comment:
```suggestion
  meta = {
    description = "SkEditor is an app for editing Skript files";
    homepage = "https://github.com/SkEditorTeam/SkEditor";
    changelog = "https://github.com/SkEditorTeam/SkEditor/releases/tag/v${version}";
    licence = lib.licenses.mit;
    maintainers = with lib.maintainers; [ eveeifyeve ];
```

Reply:
You need to update this suggestion @Aleksanaa,

Reply:
> You need to update this suggestion @Aleksanaa,

We don't work this way. At the end you still have to squash in your local branch.